sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -233,6 +233,10 @@ class JobStatusResult(object):  # pylint: disable=useless-object-inheritance, to
     :ivar int documents_not_yet_started_count: Number of documents that have not yet started being translated.
     :ivar int documents_cancelled_count: Number of documents that were cancelled for translation.
     :ivar int total_characters_charged: Total characters charged across all documents within the job.
+    :ivar bool has_completed: boolean to check whether a document finished translation or not.
```

Comment:
not sure if you meant to keep this first sentence?

Reply:
i think this explains the attribute in a nutshell
then after that, we have more detailed explanation

Reply:
This is for the job, though. Do you mean to say "whether the translation job finished or not"?

Reply:
ok will remove it

Reply:
sorry will correct it!

Reply:
so! is it ok now?

Reply:
yes, thank you! going to approve and merge!